package.json

```diff
@@ -112,6 +112,7 @@
     "d3-zoom": "^3.0.0",
     "flux": "~4.0.1",
     "fs-extra": "^10.0.0",
+    "fsevents": "^2.3.3",
```

Comment:
Unfortunately we cannot check in fsevents as a package requirement. Please back this out and roll back package-lock.json.